azure-pipelines.yml

```diff
@@ -18,6 +21,7 @@ pr:
       - feature/*
       - task/*
       - fix/*
+      - interns/*
```

Comment:
@AyuAi let me know if there are any other branch-naming conventions we will be using for interns? I think it's just `interns/` prefix on all our branches right? 

FYI this means the build will be triggered for each PR and push to a branch with the `interns/` prefix

Reply:
@acchiang Everything should start with interns/* except for our deploys to “production,” which we will use “dev” for ^^

Basically, the pipeline is our branches -> interns/dev -> dev. Though, we wont merge into interns/dev unless it passes the checks, so interns/dev -> devs should theoretically have no more checks, right?

I think it wouldn’t hurt to add dev to this list just incase 👀

Reply:
Don't worry, dev is on line 20 (it was already in the original azure_pipeline.yml file)!